### ISSUE TITLE
Install dependencies from package.json

### ISSUE THREAD

**Issue by nihey** [IonicaBizau/gpm#2]
I think it is a great way to increase the project usefulness. A lot of projects manage dependencies with a package.json file.

**Comment by IonicaBizau** [comment]
Hey!

You mean from a local project, right? Like you do `npm i`.
That's a good point! :+1:

It would be nice to run `gpm -i .`  inside of a local project. Would you like to contribute?

**Comment by nihey** [comment]
> You mean from a local project, right? Like you do `npm i`

Exactly, maybe a `gpm -i` would suit this project standarts.

> Would you like to contribute?

Sure! I can look after that on this weekend.

**Comment by IonicaBizau** [comment]
@nihey Problem with `gpm -i` is that currently [`clp`](https://github.com/IonicaBizau/node-clp) doesn't support that. By passing a `.` after `-i` (which can be handled in the bin script), you can tell the `clp` to install the current dependencies.

Something tells me you could do something like this [here](https://github.com/IonicaBizau/node-gpm/blob/master/bin/gpm#L60):

``` js
if (installOpt.value === ".") {
    pack.pack = ReadJson("package.json"); // do is async, but the idea is the same
}

// install
```

Maybe some [changes should be made](https://github.com/IonicaBizau/node-gpm/blob/master/lib/index.js#L184) (in the `getPack` you should check if the `pack` field is set already and if it is so, just callback it), but anyway, you can force `clp` to set the `package.json. :sparkles:

**Comment by IonicaBizau** [comment]
:ship: 1.1.0 -- Fixed via #4. Thanks @nihey! :cake:
